### PR TITLE
Add python-amqp to Fedora 23.

### PR DESCRIPTION
**Pull request by bowlofeggs**
This commit brings python-amqp from the pulp repo into this repo,
and updates the spec to avoid conditional statements and to use the
newer spec macros for Python and licensing.

### CODE REVIEW

File: rpms/python-amqp/python-amqp.spec [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+
+
+Name:           python-%{srcname}
+Version:        1.4.9
+Release:        2%{?dist}
+
+License:        LGPLv2+
+Summary:        Low-level AMQP client for Python (fork of amqplib)
+Group:          Development/Languages
+URL:            http://pypi.python.org/pypi/amqp
```
Comment (jeremycline) [inline rpms/python-amqp/python-amqp.spec]: It'd be nice if this URL and the Source0 URL used HTTPS explicitly. Not a huge deal, as `pypi.python.org` redirects to HTTPS.

File: rpms/python-amqp/python-amqp.spec [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+%install
+%py2_install
+
+# Remove execute bit from example scripts (packaged as doc)
+chmod -x demo/*.py
+
+
+%files
+%license LICENSE
+%doc Changelog README.rst
```
Comment (jeremycline) [inline rpms/python-amqp/python-amqp.spec]: This should probably include AUTHORS

File: rpms/python-amqp/python-amqp.spec (hunk 1 above)
Comment (bowlofeggs) [inline rpms/python-amqp/python-amqp.spec]: Fixed.

File: rpms/python-amqp/python-amqp.spec (hunk 2 above)
Comment (bowlofeggs) [inline rpms/python-amqp/python-amqp.spec]: Fixed.

**Pull request merged**